languages/csharp/exercises/concept/dictionaries/.docs/introduction.md

```diff
@@ -0,0 +1,62 @@
+A dictionary is a collection of elements where each element comprises a key and value such that if a key is passed to a method of the ditionary its associated value is returned. It has the same role as maps or associative arrays do in other languages.
+
+A dictionary can be created as follows:
+
+```csharp
+new Dictionary<int, string>{{1, "One"}, {2, "Two"}};
+// 1 => "One", 2 => "Two"
+```
+
+Note that the key and value types are part of the definition of the dictionary.
+
+Entries can be added to the dictionary as follows:
+
+```csharp
+var numbers = new Dictionary<int, string>();
+numbers.Add(3, "Three");
+numbers.Add(4, "Four");
+// 3 => "Three", 4 => "Four"
+```
+
+To remove a key-value pair simply pass the key to the `Remove` method.
```

Comment:
This example does not introduce any new syntax, right? As such, could you remove the example? The text could become something like:

```suggestion
Once constructed, entries can be added or removed from a dictionary using its built-in methods.
```

I do wonder if maybe it is useful to add the `new Dictionary<int, string>()` way of creating a dictionary to the above example that uses the object initializer syntax.